contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -285,11 +285,14 @@ BEGIN
 		RAISE E'Could not initialize babelfish with given role name: % is not the DB owner of current database.', sa_name;
 	END IF;
 
+	EXECUTE format('CREATE ROLE securityadmin CREATEROLE INHERIT PASSWORD NULL');
```

Comment:
But a role can only grant this attribute if it itself has that attribute, hence we need to keep this attribute same as sysadmin.

Reply:
But we are internally switching to bbf_role_admin which already has create role. No ?

Reply:
Currently we are switching it to internal role but if in future we decide to use directly this, it should have the required privilege and also to explicitly reflect which priv it has and also to keep privilege wise similar to sysadmin and for future reference I have created this role with required attribute as discussed in design.

Reply:
@anju15bharti IMHO, let's not over engineer this feature and lets keep it simple for now. If such requirements comes up in future then we will revisit this. I am making this point especially in this feature it is sensitive feature in terms of security.

Reply:
Agree, but we also don't need these attributes for sysadmin but we have kept it, so following that trend I have kept this attribute to this securityadmin role, Also giving this attribute to securityadmin is not escalating any privilege because even if we consider this role alone it should have this attribute.